src/greenlight/report.clj

```diff
@@ -165,7 +166,9 @@
 (defn write-junit-results
   "Render a set of test results to a JUnit XML file."
   [report-path results options]
-  (spit report-path (junit/report results options)))
+  (let [report-file (io/file report-path)]
+    (.mkdirs (.getParentFile report-file))
```

Comment:
You could use [`io/make-parents`](https://clojure.github.io/clojure/clojure.java.io-api.html#clojure.java.io/make-parents) here.
```suggestion
    (io/make-parents report-file)
```